docs/_docs/development/how-to-develop.md

```diff
@@ -11,5 +11,5 @@ Setting up your dev environment:
 - Clone the framework's source code to your machine: `git clone git@github.com:TextureGroup/Texture.git` or `git clone https://github.com/TextureGroup/Texture.git`.
 - Run `pod install` in the directory that you cloned to.
 - Open "AsyncDisplayKit.xcworkspace" file CocoaPods has just generated. The workspace includes all the source code, as well as our test suite.
-- Run `./build.sh all` locally and ensure all tests pass. Also make sure you're running the same Xcode version and Cocoapods version as the CI (currently Xcode 10.2.1 and Cocoapods 1.6). [xcversion](https://github.com/xcpretty/xcode-install) is a handy tool for keeping multiple versions of Xcode installed. You'll also need the correct [simulator device configuration](https://github.com/TextureGroup/Texture/blob/32a2ebf49b797b0ba2a74f2af44457a9aa7b1160/build.sh#L3) available.
+- Run `./build.sh all` locally and ensure all tests pass. Also make sure you're running the same Xcode version and Cocoapods version as the CI (currently Xcode 10.2.1, Cocoapods 1.6 and Carthage 0.33.0). [xcversion](https://github.com/xcpretty/xcode-install) is a handy tool for keeping multiple versions of Xcode installed. You'll also need the correct [simulator device configuration](https://github.com/TextureGroup/Texture/blob/32a2ebf49b797b0ba2a74f2af44457a9aa7b1160/build.sh#L3) available.
```

Comment:
Given that this sentence is about our CI setup which doesn't use Carthage, I wonder if we should instead mention it in the last sentence?

```suggestion
- Run `./build.sh all` locally and ensure all tests pass. Also make sure you're running the same Xcode version and Cocoapods version as the CI (currently Xcode 10.2.1 and Cocoapods 1.6). [xcversion](https://github.com/xcpretty/xcode-install) is a handy tool for keeping multiple versions of Xcode installed. You'll also need Carthage 0.33.0 and the correct [simulator device configuration](https://github.com/TextureGroup/Texture/blob/32a2ebf49b797b0ba2a74f2af44457a9aa7b1160/build.sh#L3) available.
```

Reply:
On a second thought, the CI has Carthage installed too.

```suggestion
- Run `./build.sh all` locally and ensure all tests pass. Also make sure you're running the same Xcode, Cocoapods and Carthage versions as the CI (currently Xcode 10.2.1, Cocoapods 1.6 and Carthage 0.33.0). [xcversion](https://github.com/xcpretty/xcode-install) is a handy tool for keeping multiple versions of Xcode installed. You'll also need the correct [simulator device configuration](https://github.com/TextureGroup/Texture/blob/32a2ebf49b797b0ba2a74f2af44457a9aa7b1160/build.sh#L3) available.
```